product_base_mass_add/i18n/fr.po

```diff
@@ -0,0 +1,82 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* quick_purchase
```

Comment:
I think this file is wrong. (reference to quick_purchase in all the doc). Should be regenerated.